pkg/dashboards/istio/istio_workload.go

```diff
@@ -0,0 +1,220 @@
+package istio
+
+import (
+	"github.com/perses/community-dashboards/pkg/dashboards"
+	panels "github.com/perses/community-dashboards/pkg/panels/istio"
+	"github.com/perses/community-dashboards/pkg/promql"
+	"github.com/perses/perses/go-sdk/dashboard"
+	panelgroup "github.com/perses/perses/go-sdk/panel-group"
+	listVar "github.com/perses/perses/go-sdk/variable/list-variable"
+	markdownPanel "github.com/perses/plugins/markdown/sdk/go"
+	labelValuesVar "github.com/perses/plugins/prometheus/sdk/go/variable/label-values"
+	promqlVar "github.com/perses/plugins/prometheus/sdk/go/variable/promql"
+)
+
+// Helper function to create markdown header panels
+func createWorkloadHeaderPanel(title string) panelgroup.Option {
+	return panelgroup.AddPanel(title,
+		markdownPanel.Markdown(title+" Header",
+			markdownPanel.Text("<div class=\"dashboard-header text-center\">\n<span>"+title+"</span>\n</div>"),
+		),
+	)
+}
+
+func withWorkloadGeneralSection(datasource string, labelMatcher promql.LabelMatcher) dashboard.Option {
```

Comment:
labelMatcher is not used in this function
```suggestion
func withWorkloadGeneralSection(datasource string) dashboard.Option {
```